serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,51 @@
+= Migrate Your Postgresql database with Kogito upgrade
+
+Kogito provides out of the box way to manage your database schema changes with each upgrade. Kogito uses flyway to achieve the same functionality.
+
+When you upgrade your Kogito version, by default it will pick up the latest changes available from Kogito and apply them automatically.
+
+== How to migrate
+
+.Ways to Migrate your Kogito database with Kogito and Flyway
+* Kogito provides a default mechanism for migrating your db while updating the kogito version using `quarkus.flyway.migrate-at-start` (default `true`). This would create a schema history table `flyway_schema_history` in your DB to track the version of each database, recording in it every versioned migration file applied to build that version.
```

Comment:
```suggestion
* Kogito provides a default mechanism for migrating your database while updating the Kogito version using the `quarkus.flyway.migrate-at-start` property (default value is `true`). This would create a schema history table `flyway_schema_history` in your DB to track the version of each database, recording in it every versioned migration file applied to build that version.
```